packages/apollo-tracing/src/__tests__/index.ts

```diff
@@ -1,6 +0,0 @@
-import { TracingExtension } from '../index';
```

Comment:
Glad we're removing these "tests" that are not doing anything at all.  They're not even `xit`!